src/jitlog/messages.lua

```diff
@@ -21,6 +22,37 @@ local msgs = {
     "time : timestamp",
     use_msgsize = "label",
   },
+
+  {
+    name = "traceexit",
+    "isgcexit : bool",
+    "traceid : 14",
+    "exit : 9",
+  },
+
+  {
+    name = "traceexit_large",
+    "isgcexit : bool",
+    "traceid : u16",
+    "exit : u16",
+  },
+
+  {
+    name = "alltraceflush",
+    "reason : u16",
+    "time : timestamp",
+    "tracelimit : u16",
+    "mcodelimit : u32",
+  },
+
+  {
+    name = "gcstate",
+    "state : 8",
+    "prevstate : 8",
+    "totalmem : u32",
+    "strnum : u32",
+    "time : timestamp",
+  },
```

Comment:
i guess this is still in flux, but it would be nice to have common fields at the same offsets.

Reply:
this is still readable, but if a user (a developer embedding LuaJIT in some application) is supposed to be able to write these message types, this language will need some documentation.

Reply:
Added a bit of documentation to the top of the file

Reply:
I initially started out trying to keep fields to there natural alignment when deciding there order but that doesn't really mater with 1 byte struct packing setup and the message offset in the buffer probably won't a multiple of 4.  i moved all time stamp fields to right after the message header but there offset will still vary between fixed and variable sized messages.

Reply:
I think Javier's comment isn't about the ordering from the machine perspective, but having things in the same order makes it easier for humans to read. I think that's a sensible suggestion and, unless there's a clear performance penalty, we should do so.

Reply:
I did reorder them but was just pointing the limitations.

Reply:
So they're now all in the same human-friendly order?

Reply:
well i reordered them based on message layout first but but i left fields that still go before them automatically there in the declarations i guess can move to them to the very first

Reply:
I'm confused. Can you answer my question with "yes" or "no" please.

Reply:
the answer was kind of no but i've made them truly first now

Reply:
OK, thanks.